bundle/edu.gemini.catalog/src/main/java/jsky/catalog/AbstractCatalogDirectory.java

```diff
@@ -582,7 +549,7 @@ private TreeModelEvent _getTreeModelEvent(Catalog cat) {
         int n = l.size();
         Catalog[] ar = new Catalog[n];
         for (int i = 0; i < n; i++)
-            ar[n - i - 1] = (Catalog) l.get(i);
+            ar[n - i - 1] = l.get(i);
```

Comment:
The cleanup was mostly about getting rid of compiler warnings, making the code proper may be too big an enterprise if we are going to delete the skycat code.